src/V3EmitCImp.cpp

```diff
@@ -610,6 +613,66 @@ class EmitCTrace final : EmitCFunc {
         puts("\n");
     }
 
+    V3OutCFile* typesFp() const VL_MT_SAFE { return m_typesFp; }
+
+    void openNextTypesFile() {
+        UASSERT(!m_typesFp, "Declarations output file already open");
+
+        string filename
+            = (v3Global.opt.makeDir() + "/" + topClassName() + "_" + protect("_Trace_Decls"));
+        filename = m_uniqueNames.get(filename);
+        filename += "__Slow.cpp";
+
+        AstCFile* const cfilep = createCFile(filename, m_slow, true /*source*/);
+        cfilep->support(true);
+        m_cfilesr.push_back(cfilep);
+
+        if (optSystemC()) {
+            m_typesFp = new V3OutScFile{filename};
+        } else {
+            m_typesFp = new V3OutCFile{filename};
+        }
+        typesFp()->putsHeader();
+        typesFp()->puts("// DESCR"
+                        "IPTION: Verilator output: Tracing declarations\n");
+
+        // Includes
+        typesFp()->puts("#include \"" + v3Global.opt.traceSourceLang() + ".h\"\n");
+        typesFp()->puts("\n");
+
+        typesFp()->puts("\nvoid " + prefixNameProtect(m_modp) + "__"
+                        + protect("trace_decl_types_sub" + cvtToStr(m_traceTypeSubs++)) + "("
```

Comment:
```suggestion
                        + protect("traceDeclTypesSub" + cvtToStr(m_traceTypeSubs++)) + "("
```

Favor camel caps style.